### ISSUE TITLE
remove recommendation to use aget for JS object property access

### ISSUE THREAD

**Issue by oakmac** [oakmac/cljs-cheatsheet#15]
re: https://twitter.com/mfikes/status/882013594828800000

Should recommend using [cljs-oops](https://github.com/binaryage/cljs-oops) for JS object property access.

**Comment by jiacai2050** [comment]
https://clojurescript.org/news/2017-07-14-checked-array-access

How is this issue going?
I think we need to remove this usage ASAP to avoid misguiding newbies.

**Comment by oakmac** [comment]
I agree. I have known about this misinformation on the cheatsheet for some
time now. Hopefully will have some time for a fix tonight.

**Comment by oakmac** [comment]
This is fixed now:

![screen shot 2018-01-13 at 4 44 07 pm](https://user-images.githubusercontent.com/98485/34910352-017bec60-f881-11e7-8f31-993e8ba91146.png)

**Comment by jiacai2050** [comment]
Great, I think we can also add goog.object/get, goog.object/set, goog.object/getValueByKeys, and put cljs-oops in External Library section
